common/src/jni/main/cpp/NativeCrypto.cpp

```diff
@@ -5676,16 +5676,19 @@ static int new_session_callback(SSL* ssl, SSL_SESSION* session) {
 
     jobject sslHandshakeCallbacks = appData->sslHandshakeCallbacks;
     jclass cls = env->GetObjectClass(sslHandshakeCallbacks);
-    jmethodID methodID = env->GetMethodID(cls, "onNewSessionEstablished", "(J)Z");
+    jmethodID methodID = env->GetMethodID(cls, "onNewSessionEstablished", "(J)V");
     JNI_TRACE("ssl=%p new_session_callback calling onNewSessionEstablished", ssl);
-    jboolean ret = env->CallBooleanMethod(sslHandshakeCallbacks, methodID,
+    env->CallBooleanMethod(sslHandshakeCallbacks, methodID,
                                           reinterpret_cast<jlong>(session));
```

Comment:
Super nit-picky nitpick: formatting? (I just have a key-binding for clang-format and try not to think too hard about such things.)